app/models/feed.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+class Feed < ActiveRecord::Base
+  has_and_belongs_to_many :pages
+
+  validates :name, presence: true, uniqueness: true
+  validates :url, presence: true, uniqueness: true
+
+  acts_as_url :name, url_attribute: :slug, only_when_blank: true, allow_duplicates: false
```

Comment:
Fair enough. Let's leave slug there for now, it's good to have in case we need it, which there is a fair chance of, and if not we can easily drop it later.